src/main/python/mmlspark/vw/VowpalWabbitRegressor.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+from mmlspark.vw._VowpalWabbitRegressor import _VowpalWabbitRegressor, _VowpalWabbitRegressorModel
+from pyspark.ml.common import inherit_doc
+
+@inherit_doc
+class VowpalWabbitRegressor(_VowpalWabbitRegressor):
+    def _create_model(self, java_model):
+        model = VowpalWabbitRegressorModel()
+        model._java_obj = java_model
+        model._transfer_params_from_java()
+        return model
+
+@inherit_doc
+class VowpalWabbitRegressorModel(_VowpalWabbitRegressorModel):
```

Comment:
maybe rename to RegressionModel to be consistent with other models?  I believe most are regression instead of regressor, worth double-checking

Reply:
https://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.ml.classification.LogisticRegressionModel
https://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.ml.regression.LinearRegressionModel

Reply:
really good catch!